openedx/adg/lms/webinars/helpers.py

```diff
@@ -30,7 +33,7 @@ def send_webinar_emails(template_slug, webinar, recipient_emails, send_at=None):
         'webinar_id': webinar.id,
         'webinar_title': webinar.title,
         'webinar_description': webinar.description,
-        'webinar_start_time': webinar.start_time.strftime("%B %d, %Y %I:%M %p %Z")
+        'webinar_start_time': webinar.start_time.astimezone(pytz.timezone(WEBINAR_DEFAULT_TIME_ZONE)).strftime("%B %d, %Y %I:%M %p %Z")
```

Comment:
Why can't we use `WEBINARS_TIME_FORMAT` here?

Reply:
The change was already there and i don't have any context about why the last dev didn't used the constant here as this change is in master . if you want i can make this change in context of my ticket too.

Reply:
Yes, let's please

Reply:
Made the changes in each section where we are using the timezone and format